metrics/metrics.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2017, Square, Inc.
+
+// Package metrics is a wrapper of github.com/rcrowley/go-metrics.
```

Comment:
At the risk of being pedantic :-) I'd rephrase this. This doc line is supposed to say what the pkg provides. Being a wrapper is what the pkg _is_. It's also an implementation detail. For example, our next intern will come across this and wonder what this package does (i.e. provides). (Then after, they might want to know how it does that, but that's where the code itself comes in.)

Reply:
I see where you're coming from. There's a difference between what the package is vs what is its purpose.

What do you think of the following?

Package metrics provides a threadsafe way to manage counters and timers.